packages/terra-clinical-result/src/flowsheetresultcell/FlowsheetResultCell.jsx

```diff
@@ -0,0 +1,225 @@
+import ClinicalResult from '../ClinicalResult';
+import ClinicalResultBloodPressure from '../ClinicalResultBloodPressure';
+import observationPropShape from '../proptypes/observationPropTypes';
+import ResultError from '../common/other/_ResultError';
+import NoData from '../common/other/_KnownNoData';
+import styles from './FlowsheetResultCell.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   *  A set of clinical results.                                                          .
+   */
+  resultDataSet: PropTypes.arrayOf(PropTypes.shape({
+    /**
+     *  A single clinical result or blood pressure result id.
+     */
+    id: PropTypes.string,
+    /**
+     *  A single clinical result or blood pressure result.
+     */
+    resultData: observationPropShape,
+  })),
+  /**
+   * Visually hides the unit of measure when presented in a series of side-by-side columns of the same unit.
+   */
+  hideUnit: PropTypes.bool,
+  /**
+   * The padding styling to apply to the Time Column Header Cell.
+   * One of `'none'`, `'standard'`, `'compact'`.
+   */
+  paddingStyle: PropTypes.oneOf(['none', 'standard', 'compact']),
+  /**
+   * Override that shows an Error display. Used when there is a known error or problem when retrieving or assembling the clinical result data.
+   */
+  hasResultError: PropTypes.bool,
+  /**
+   * Override that shows a known "No Data" display. Used when there is known to be no value for a given clinical result concept at a specific datetime.
+   */
+  hasResultNoData: PropTypes.bool,
+};
+
+const defaultProps = {
+  resultDataSet: [],
+  hideUnit: false,
+  paddingStyle: 'compact',
+  hasResultError: false,
+  hasResultNoData: false,
+};
+
+const isEmpty = (str) => (!str || str.length === 0);
+
+const FlowsheetResultCell = (props) => {
+  const {
+    resultDataSet,
+    hideUnit,
+    paddingStyle,
+    hasResultError,
+    hasResultNoData,
+    ...customProps
+  } = props;
+
+  let flowsheetResultCellDisplay = null;
+
+  if (hasResultError || hasResultNoData) {
+    flowsheetResultCellDisplay = hasResultError ? (<div className={cx('single-result-display')}><ResultError /></div>) : (<div className={cx('single-result-display')}><NoData /></div>);
+  } else {
+    const createResultsDisplay = (resultSet) => {
+      let resultKeyID;
+      let resultsDisplay = [];
+      let resultsInnerDisplay;
+      let additionalResultCount = 0;
+      let additionalResultInnerDisplay;
+      const additionalResultInterpretations = [];
+      let additionalResultInterpretationIndicator;
+      let endAccessoryIcons;
+      let singleResultHasComment = false;
+      let singleResultIsModified = false;
+      let singleResultIsUnverified = false;
+
+      if (!resultSet || !resultSet.length) {
+        resultKeyID = 'Error';
+        resultsDisplay.push(<div key={(`ClinicalResultDisplay-${resultKeyID}`)} className={cx('single-result-display ')}><ResultError /></div>);
```

Comment:
Could there ever be multiple error results displayed? If yes, the key won't be unique and react will throw a warning on the dom